delivery_gls_asm/models/gls_asm_request.py

```diff
@@ -0,0 +1,481 @@
+                    </Horarios>
+                    <RecogerEn>
+                        <Nombre>{remite_nombre}</Nombre>
+                        <Direccion>{remite_direccion}</Direccion>
+                        <Poblacion>{remite_poblacion}</Poblacion>
+                        <Provincia>{remite_provincia}</Provincia>
+                        <Pais>{remite_pais}</Pais>
+                        <CP>{remite_cp}</CP>
+                        <Telefono>{remite_telefono}</Telefono>
+                        <Movil>{remite_movil}</Movil>
+                        <Email>{remite_email}</Email>
+                        <Contacto></Contacto>
+                    </RecogerEn>
+                    <Entregas>
+                        <Envio>
+                            <FechaPrevistaEntrega>{fechaentrega}</FechaPrevistaEntrega>
+                            <Portes>{portes}</Portes>
+                            <Servicio>{servicio}</Servicio>
+                            <Horario>{horario}</Horario>
+                            <Bultos>{bultos}</Bultos>
+                            <Peso>{peso}</Peso>
+                            <Destinatario>
+                                <Nombre>{destinatario_nombre}</Nombre>
+                                <Direccion>{destinatario_direccion}</Direccion>
+                                <Poblacion>{destinatario_poblacion}</Poblacion>
+                                <Provincia>{destinatario_provincia}</Provincia>
+                                <Pais>{destinatario_pais}</Pais>
+                                <CP>{destinatario_cp}</CP>
+                                <Telefono>{destinatario_telefono}</Telefono>
+                                <Movil>{destinatario_movil}</Movil>
+                                <Email>{destinatario_email}</Email>
+                                <Observaciones>{observaciones}</Observaciones>
+                            </Destinatario>
+                        </Envio>
+                    </Entregas>
+                    <Referencias>
+                        <Referencia tipo="C">{referencia_c}</Referencia>
+                        <Referencia tipo="A">{referencia_a}</Referencia>
+                    </Referencias>
+                </Recogida>
+            </Servicios>
+        """.format(
+            **kwargs
+        )
+
+    def _send_shipping(self, vals):
+        """Create new shipment
+        :params vals dict of needed values
+        :returns dict with GLS response containing the shipping codes, labels,
+        an other relevant data
+        """
+        vals.update({"uidcustomer": self.uidcustomer})
+        xml = Raw(self._prepare_send_shipping_docin(**vals))
+        _logger.debug(xml)
+        try:
+            res = self.client.service.GrabaServicios(docIn=xml)
+        except Exception as e:
+            raise UserError(
+                _(
+                    "No response from server recording GLS delivery {}.\n"
+                    "Traceback:\n{}"
```

Comment:
Estas cadenas tendrían el mismo problema de traducibilidad en el orden de los argumentos.

@moylop260 parece que estos casos no los detecta pylint-odoo.

Reply:
Fixed from https://github.com/OCA/pylint-odoo/pull/395

Thank you!

Reply:
Gracias!

Reply:
FYI odoo >= 14.0 supports using `_("%(arg1)s", arg1="value1")` in order to use lazy formatting

More info about:
 - https://github.com/OCA/pylint-odoo/issues/378

Code related:
 - https://github.com/odoo/odoo/blob/6a6e5f92789e1c/odoo/tools/translate.py#L446
   - Notice it only support binop `%` and it is not supporting `.format`


It is similar to `_logger.info('Hello, {}', 'world')` raises the following error:

```txt
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-19-d7036f474fad>", line 1, in <module>
    _logger.info('Hello, {}', 'world')
Message: 'Hello, {}'
Arguments: ('world',)
```

Reply:
Gracias por la info